block/manager_test.go

```diff
@@ -20,11 +20,13 @@ import (
 )
 
 func TestInitialState(t *testing.T) {
```

Comment:
This is good to update this test, but we should also have a low level unit test of `getInitialState` that can easily target each code path of the function. 

This low level unit test would then be able to verify we are exiting the function where we expect to, with or without the expect error. 

`TestInitialState` can't verify that as it is just calling `NewManager`